### ISSUE TITLE
The UI is allowing me to look at opponent's foretell cards

### ISSUE THREAD

**Issue by foryayfan** [magefree/mage#7640]
I was testing a deck against the AI opponent. The AI opponent put several cards in exile as foretell cards. The UI opens a window for each foretold card. I can click on the cards in those windows and it reveals the name of the cards that the opponent has foretold. A player should not be able to see the card names of the opponents foretold cards.
![java_4DFNcmgprL](https://user-images.githubusercontent.com/75407590/109793835-2088db80-7c50-11eb-9d65-465d709d7720.jpg)

**Comment by jeffwadsworth** [comment]
Are you trying this with the latest client?  Also, did you actually click the "Yes, look at the card" and see the card?  Your picture just shows the option.

**Comment by foryayfan** [comment]
Yes I am using the latest client. Yes I could view the face of the card. I can provide one additional detail: those cards were foretold using the Dream Devourer ability. Perhaps that is why I was allowed to view the card face.

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by jeffwadsworth** [comment]
The controller of the source ability (the Foretold Special Ability) is being changed to the current priority player now for some reason.  Thus, when it checks the permission, it is allowed to view the card in exile.

**Comment by jeffwadsworth** [comment]
This appears to be working correctly now.  Please open again if it resurfaces.

**Comment by xenohedron** [comment]
reported again today in a REMA draft:
"bug with foretell card - I was able to look at opponents card. I also had the button to cast it - but that did nothing"
